src/main/java/com/toptrumps/db/IndividualGameDAOImpl.java

```diff
@@ -1,44 +1,47 @@
 package com.toptrumps.db;
 import com.toptrumps.core.statistics.GameStateCollector;
-
+import com.toptrumps.core.player.Player;
 import java.sql.*;
-import java.sql.Connection; 
-import java.sql.SQLException; 
-import java.sql.Statement; 
-
-public class IndividualGameDAOImpl implements IndividualGameDAO{
-
-    public IndividualGameDAOImpl(){
-
-    }
-
-    // TODO: Update the DAO to get data from GameStateCollector
-    public boolean create(GameStateCollector gameState){
-        Connection con = ConnectionFactory.getConnection();
-        PreparedStatement createRow = null;
-        String createRowSQL = "INSERT INTO individual_game_data(winner,rounds_played,draws)" +
-        "VALUES (?,?,?);";
 
-        try {
+    public class IndividualGameDAOImpl implements GameDAO{
 
-            createRow = con.prepareStatement(createRowSQL);
-            createRow.setString(1,"Human");
-            createRow.setInt(2,20);
-            createRow.setInt(3,20);
-            createRow.executeUpdate();
-
-        }catch(SQLException e){
-            e.printStackTrace();
+        public IndividualGameDAOImpl(GameStateCollector gameState){
```

Comment:
I still think GameDAOImpl would be enough :)

Reply:
💯 💯 💯 can do